convert_kinbody_to_urdf.py

```diff
@@ -0,0 +1,194 @@
+# convert_kinbody_to_urdf.py
+
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+
+class Kinbody_Converter:
+    def __init__(self):
+        self.kinbody = None
+        self.urdf = None
+        self.link = None
```

Comment:
`self.link` represents the `link` element right under the root (`robot`). Since it was not clean to access the `link` element through `self.urdf` -- it should be `self.urdf.iter('link')[0]`, I kept it as a member variable.